lib/components/app_drawer.dart

```diff
@@ -41,6 +41,17 @@ class AppDrawer extends StatelessWidget {
               child: const Icon(Icons.add),
             ),
           ),
+          Align (
+            alignment: Alignment.topRight,
+            child: FloatingActionButton(
+              onPressed: () {
+                appData.removeTank(appData.currentTank);
+                appData.currentTank = Tank("", "");
```

Comment:
How many places in the code do we have `Tank("", "")`? How would we remove that duplicate code? (This should probably be a separate PR, but is an example of repetition that could be simplified.)

Reply:
From what I can find, it occurs 6 times in the app and once in testing. I should be able to write a short function to set it to a default empty Tank.